### PR TITLE
new default intervals for Pilot

### PR DESCRIPTION
**Pull request by rshriram**
Repeat of PR #779 
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rshriram** [comment]
Verified from logs that the short refresh delays are maintained for tests..
```
    "cds": {
W0913 14:39:45.827]       "cluster": {
W0913 14:39:45.827]         "name": "cds",
W0913 14:39:45.827]         "connect_timeout_ms": 1000,
W0913 14:39:45.827]         "type": "strict_dns",
W0913 14:39:45.827]         "lb_type": "round_robin",
W0913 14:39:45.827]         "hosts": [
W0913 14:39:45.827]           {
W0913 14:39:45.827]             "url": "tcp://istio-pilot.mixer-test-83a972637e3b4521955f6fce48:8080"
W0913 14:39:45.827]           }
W0913 14:39:45.827]         ]
W0913 14:39:45.827]       },
W0913 14:39:45.828]       "refresh_delay_ms": 1000
W0913 14:39:45.828]     }
W0913 14:39:45.828]   },
```

File: install/kubernetes/istio-auth.yaml [hunk 1]
```diff
@@ -554,21 +554,62 @@ data:
...
-          drainDuration:          2s
-          parentShutdownDuration: 3s
-          discoveryAddress:       istio-pilot.istio-system:8080
-          discoveryRefreshDelay:  1s
-          connectTimeout:         1s
-          proxyAdminPort:         15000
-          zipkinAddress:          zipkin.istio-system:9411
-          statsdUdpAddress: istio-mixer.istio-system:9125
+      # See rdsRefreshDelay for explanation about this setting.
+      discoveryRefreshDelay: 30s
```
Comment (linsun) [inline install/kubernetes/istio-auth.yaml]: Did you intentionally remove the indent under defaultConfig?  Seems it reads better with the indent.

Comment (rshriram) [reply install/kubernetes/istio-auth.yaml]: This seems to be the indent that emacs gave me in YAML mode.

**Pull request merged**